openlr_dereferencer/decoding/candidates.py

```diff
@@ -86,8 +94,9 @@ def match_tail(
 
     If not, a `LRDecodeError` exception is raised."""
     last_lrp = len(tail) == 1
-    # The maximum accepted length. This helps A* to save computational time
-    maxlen = (1 + MAX_DNP_DEVIATION) * current.dnp
+    # The accepted distance to next point. This helps to save computations and filter bad paths
+    minlen = (1 - MAX_DNP_DEVIATION) * current.dnp - TOLERATED_DNP_DEV
```

Comment:
This could potentially fall below `0`. Maybe we can set to `0` in that case.

`max(0, (1 - MAX_DNP_DEVIATION) * current.dnp - TOLERATED_DNP_DEV)`

Reply:
True, but as falling below 0 would have no undesirable effect, I would _prefer_ to not change, preserving all readability.